src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -1010,7 +1010,7 @@ private long fetchLatestCommittedOffsetFromSnowflake() {
       offsetToken = this.channel.getLatestCommittedOffsetToken();
       LOGGER.info(
           "Fetched offsetToken for channelName:{}, offset:{}", this.getChannelName(), offsetToken);
-      return offsetToken == null
+      return offsetToken == null || offsetToken.equalsIgnoreCase("NULL")
```

Comment:
this is a behavior change, right?

Reply:
That's a good point but I dont know how else we would identify NULL set by our sys function. Somehow I wasnt able to get a null offsetToken (like a new channel) after setting in sys function. (I tried NULL, null, empty strings but with no luck)